packages/block-library/src/gallery/edit.js

```diff
@@ -227,7 +237,7 @@ class GalleryEdit extends Component {
 	componentDidMount() {
 		const { attributes, mediaUpload } = this.props;
 		const { images } = attributes;
-		if ( every( images, ( { url } ) => isBlobURL( url ) ) ) {
+		if ( images.length && every( images, ( { url } ) => isBlobURL( url ) ) ) {
```

Comment:
as we [agreed](https://github.com/WordPress/gutenberg/pull/18265#discussion_r354505541) let's add back platform check for now